SonarQube.TeamBuild.PostProcessor/MSBuildPostProcessor.cs

```diff
@@ -0,0 +1,147 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="MSBuildPostProcessor.cs" company="SonarSource SA and Microsoft Corporation">
```

Comment:
This class contains almost all of that was in Program.cs. The only changes are to inject objects in the constructor rather than having them created inline.